bundle_result.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package bundle_result;
+
+// Indicates the bundle successfully won its auction and was dispatched to the current leader.
+message Dispatched {
+  // The bundle's placement within the auction upon completion (e.g. 1st or 2nd place).
+  uint32 auction_placement = 1;
+
+  // This bundle's bid.
+  uint64 originator_bid_lamports = 2;
+
+  // Window of slots this bundle was sent for.
+  repeated uint64 slot_window = 4;
+
+  // List of validators this bundle was dispatched to.
+  repeated string validator_identities = 5;
```

Comment:
does this store pubkeys or ips?

Reply:
referring to pubkey using the terminology from [this](https://docs.solana.com/running-validator/vote-accounts#validator-identity)

Reply:
gotcha, thanks for the link. maybe update the comment to `  // List of validator pubkeys this bundle was dispatched to.` ?